File: SDKs/github.com/api.github.com/1.1.4/python/src/sdk/models/operations/orgs_list_pending_invitations.py
from dataclasses import dataclass, field
from typing import List,Optional
from sdk.models import shared


@dataclass
class OrgsListPendingInvitationsPathParams:
    org: str = field(metadata={'path_param': { 'field_name': 'org', 'style': 'simple', 'explode': False }})
    

@dataclass
class OrgsListPendingInvitationsQueryParams:
    page: Optional[int] = field(default=None, metadata={'query_param': { 'field_name': 'page', 'style': 'form', 'explode': True }})
    per_page: Optional[int] = field(default=None, metadata={'query_param': { 'field_name': 'per_page', 'style': 'form', 'explode': True }})
    

@dataclass
class OrgsListPendingInvitationsRequest:
    path_params: OrgsListPendingInvitationsPathParams = field()
    query_params: OrgsListPendingInvitationsQueryParams = field()
    

@dataclass
class OrgsListPendingInvitationsResponse:
    content_type: str = field()
    headers: dict[str, List[str]] = field()
    status_code: int = field()
    basic_error: Optional[shared.BasicError] = field(default=None)
    organization_invitations: Optional[List[shared.OrganizationInvitation]] = field(default=None)
    

File: SDKs/github.com/api.github.com/1.1.4/go/pkg/models/operations/orgslistpendinginvitations.go
package operations

import (
	"openapi/pkg/models/shared"
)

type OrgsListPendingInvitationsPathParams struct {
	Org string `pathParam:"style=simple,explode=false,name=org"`
}

type OrgsListPendingInvitationsQueryParams struct {
	Page    *int64 `queryParam:"style=form,explode=true,name=page"`
	PerPage *int64 `queryParam:"style=form,explode=true,name=per_page"`
}

type OrgsListPendingInvitationsRequest struct {
	PathParams  OrgsListPendingInvitationsPathParams
	QueryParams OrgsListPendingInvitationsQueryParams
}

type OrgsListPendingInvitationsResponse struct {
	ContentType             string
	Headers                 map[string][]string
	StatusCode              int64
	BasicError              *shared.BasicError
	OrganizationInvitations []shared.OrganizationInvitation
}
